packages/react-native/React/CoreModules/RCTDevMenu.mm

```diff
@@ -299,6 +299,9 @@ - (void)setDefaultJSBundle
                            devSettings.isHotLoadingEnabled = !devSettings.isHotLoadingEnabled;
                          }]];
   }
+  
+  RCTDevMenuItem *perfMonitorItem = [(id<DevMenuItemFactory>)[_moduleRegistry moduleForName:"PerfMonitor"] devMenuItem];
```

Comment:
Why not just casting it to `RCTPerfMonitor *`? I would rather avoid introducing new types in the public APIs that we then need to maintain...

Reply:
@cipolleschi This requires RCTPerfMonitor to expose it's interface: https://github.com/facebook/react-native/blob/9ba56f6f5d0601b29c68390600ce0ab2152904ae/packages/react-native/React/CoreModules/RCTPerfMonitor.mm#L63

Should I move it to `CoreModulesHeaders`?

Reply:
what if:
```objc
id perfMonitorItemOpaque = [_moduleRegistry moduleForName:"PerfMonitor"];
if ([perfMonitorItemOpaque respondsToSelector:SEL(@"devMenuItem")]) {
  RCTDevMenuItem *perfMonitorItem = [perfMonitorItemOpaque performSelector:#SEL(@"devMenuItem")];
  // add the item
}
```

Reply:
@cipolleschi Fixed